pypicloud/views/api.py

```diff
@@ -116,12 +117,12 @@ def download_package(context, request):
             public=True, max_age=request.registry.package_max_age
         )
         request.response.headers.update(cache_control)
-        request.response.body = data
+        request.response.body_file = data_fp
         request.response.content_type = "application/octet-stream"
         return request.response
     if request.registry.stream_files:
-        with request.db.storage.open(package) as data:
-            request.response.body = data.read()
+        data_fp = request.db.storage.open(package)  # will be closed by garbage collector
+        request.response.body_file = data_fp
```

Comment:
looks like the [current](https://github.com/stevearc/pypicloud/blob/1.3.5/pypicloud/storage/object_store.py#L119) `open` implementation:
- won't accept removing the `with`
- will read the whole blob into memory anyway

using `smart_open` will solve this